evals/README.md

```
@@ -64,14 +64,52 @@ Caveat: while the harness is running, the live agent view is using the override
 
 ## Comparing against a baseline
 
+`npm run eval` automatically compares each run against the blessed baseline for the active `(provider, model)` and prints a regressions-only summary at the end:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)